packages/next/src/build/index.ts

```diff
@@ -2771,14 +2790,15 @@ export default async function build(
             await fs.unlink(serverBundle)
           }
 
-          staticPaths.forEach((routes, originalAppPath) => {
+          staticPaths.forEach((prerenderedRoutes, originalAppPath) => {
             const page = appNormalizedPaths.get(originalAppPath) || ''
             const appConfig = appDefaultConfigs.get(originalAppPath) || {}
-            let hasDynamicData =
+
+            let hasRevalidateZero =
```

Comment:
nit: I found the original name a bit clearer since we use a revalidation of 0 synonymously with dynamic access. The significance of this check becomes lost and it creates a bit of confusion with later comments (ie, right below, "_if the page was marked as being static but it contains dynamic data..._"

Reply:
In a PPR world though, we don't actually use the `revalidate: 0` to indicate if a page has dynamic data or not, we simply postpone. Renaming this here adds additional clarity so lower in the code it's clearer what we're comparing directly.